src/console.c

```diff
@@ -97,30 +97,21 @@ void console_init() {
     console_putstring("\nconsole: initialized\n");
 }
 
-uint8_t console_verify_color_range(uint8_t x) {
-    if (x < 0) {
-        return 0;
-    }
-    if (x > 255) {
-        return 255;
-    }
-    return x;
-}
-
 void console_set_fgcolor(uint8_t r, uint8_t g, uint8_t b) {
-    r = console_verify_color_range(r);
-    g = console_verify_color_range(g);
-    b = console_verify_color_range(b);
+    r = graphics_verify_color_range(r);
+    g = graphics_verify_color_range(g);
+    b = graphics_verify_color_range(b);
```

Comment:
Doubles up with @kkoser 's earlier comment regarding the verification. I like the idea of the console asking graphics, but right now nothing different will ever be coming back if console_set_fg_color has uint8_t's coming in. It's 3 extra function calls that would allow us to change graphics_verify later with no impact here, but I don't see us changing our color ranges.